jmx_prometheus_javaagent/src/test/resources/test.yml

```diff
@@ -0,0 +1,33 @@
+---
+lowercaseOutputLabelNames: true
+lowercaseOutputName: true
+rules:
+- pattern: 'Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>(\w+):'
```

Comment:
We're not actually using this config, so I'd suggest keeping it minimal to avoid confusion,